packages/jellyfish-api-core/src/category/oracle.ts

```diff
@@ -74,6 +74,16 @@ export class Oracle {
     const { utxos = [] } = options
     return await this.client.call('setoracledata', [oracleid, timestamp, options.prices, utxos], 'number')
   }
+
+  /**
+   * Returns oracle data.
+   *
+   * @param {string} oracleid
+   * @return {Promise<string>} txid
+   */
+  async getOracleData (oracleid: string): Promise<string> {
```

Comment:
```suggestion
  async getOracleData (oracleid: string): Promise<OracleData?> {
```

Pretty sure it's not string